src/OgGroupAudienceHelper.php

```diff
@@ -48,4 +48,54 @@ public static function checkFieldCardinality(ContentEntityInterface $entity, $fi
     return $entity->get($field_name)->count() < $cardinality;
   }
 
+  /**
+   * Returns the first group audience field that matches the given entity.
+   *
+   * @param \Drupal\Core\Entity\ContentEntityInterface $entity
+   *   The content entity to find a matching group audience field for.
+   * @param string $group_type
+   *   The group type that should be referenced by the group audience field.
+   * @param string $group_bundle
+   *   The group bundle that should be referenced by the group audience field.
+   * @param bool $skip_access_check
+   *   Set this to TRUE to not check if the current user has access to the
+   *   field. Defaults to FALSE.
+   *
+   * @return string
+   *   The name of the group audience field.
+   */
+  public static function getBestField(ContentEntityInterface $entity, $group_type, $group_bundle, $skip_access_check = FALSE) {
```

Comment:
It is already on the `OgGroupAudience` class so I vote `getMatchingField` but let's see what @amitaibu thinks.